packages/wasi/test/hrtime.bigint.test.ts

```diff
@@ -22,11 +22,11 @@ describe("hrtime Polyfill", () => {
     // Wait for a second
     await waitForTime(1000);
     diffTime = hrtime() - start;
-    expect(diffTime > 1.0e9 && diffTime < 1.3e9).toBeTruthy();
+    expect(diffTime > 0.9e9 && diffTime < 1.4e9).toBeTruthy();
```

Comment:
Previous tests failed here: https://dev.azure.com/wasmerio/wasmer-js/_build/results?buildId=1752

That's why I loosened a bit the time